### DISCUSSION TITLE
Server Valid and not reliable for Production

### DISCUSSION THREAD

**Discussion by shoaibmalik2611** [saran-surya/email_auth_node#25] [Q&A]
I/flutter (25370): email-auth >> Initialising Email-Auth server
I/flutter (25370): email-auth >> Remote server is not available -- using test server --
I/flutter (25370): email-auth >> ❗ Warning this is not reliable on production
I/flutter (25370): email-auth >> The remote server configurations are valid
I/flutter (25370): --- Package Error ---
I/flutter (25370): Unable to access server. 😑
I/flutter (25370): --- End Package Error ---

**Comment by saran-surya** [comment]
Yes what is your remote server url, kindly post the configurations without server key

**Comment by saran-surya** [comment]
Hi @shoaibmalik2611 is your issue resolved ?
